tools/run_tests/performance/bq_upload_result.py

```diff
@@ -157,6 +169,40 @@ def _populate_metadata_inplace(scenario_result):
     scenario_result['metadata'] = metadata
 
 
+def _populate_metadata_from_file(scenario_result, test_metadata_file):
+    with open(test_metadata_file, 'r') as f:
+        test_metadata = json.loads(f.read())
+
+    """eliminate managedField from metadata set"""
```

Comment:
You should add comments with #. """""" is a special case for docstrings (right after the function header).